lib/rules/react-prefer-private-members.js

```diff
@@ -0,0 +1,85 @@
+const Components = require('eslint-plugin-react/lib/util/Components');
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Disallow public members within React component classes',
+      category: 'Possible Errors',
```

Comment:
Not really sure what to put in here. Just did my best to follow other rules in this repo.

Reply:
It's probably best practices